Dockerfile

```diff
@@ -14,8 +14,8 @@ ENV VIRTUAL_ENV=/backend
 ENV BACKEND=/backend
 
 #Install Cron
-RUN apt-get -y update
-RUN apt-get -y install libv8-dev cron gunicorn logrotate curl mydumper rsync vim
+RUN apt-get -y update && apt-get install -y pcregrep \
+  && apt-get -y install libv8-dev cron gunicorn logrotate curl mydumper rsync vim
```

Comment:
Why using `apt-get install` separately and not just include `pcregrep` at the end of list of packages